### PR TITLE
BETA: Register creed.is-a.dev

### PR DESCRIPTION
**Pull request by M-creed**
Added `creed.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by Stef-00012 — CHANGES_REQUESTED** [review]

File: domains/creed.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "M-creed",
+        "email": "marwankalosha005@gmail.com"
+    },
+    "record": {
+        "A": ["185.199.108.153"]
```
Comment (Stef-00012) [inline domains/creed.json:7]: I cannot reach your website, 404 page

(if you want to use github pages through A record, it's 4 IPs
- `185.199.111.153`
- `185.199.110.153`
- `185.199.109.153`
- `185.199.108.153`
)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Review by Stef-DP — APPROVED** [review]
seema good now

**Comment by Stef-00012** [comment]
btw fyi you put the devacrd image as hyperlink instead of as actual image

**Comment by M-creed** [comment]
> seema good now

Why the domain didn't approved yet?!

File: domains/creed.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "M-creed",
+        "email": "marwankalosha005@gmail.com"
+    },
+    "record": {
+        "CNAME": "m-creed.github.io"
```
Comment (CuteDog5695) [inline domains/creed.json:7]: cant access website (404 page)

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by CuteDog5695** [comment]
> > seema good now
> 
> Why the domain didn't approved yet?!

thats because that was a helper, not a maintainer. helpers cant merge prs, only maintainers

**Comment by CuteDog5695** [comment]
> > > > seema good now
> > > 
> > > 
> > > Why the domain didn't approved yet?!
> > 
> > 
> > thats because that was a helper, not a maintainer. helpers cant merge prs, only maintainers
> 
> Ok thank you ,am waiting for maintainer

thing is i am a maintainer, if you take a look a bit up you can see that i didnt merge it since website doesnt exist. you'll have to fix that then i can merge it

File: domains/creed.json (hunk 2 above)
Comment (Stef-00012) [reply domains/creed.json:7]: his website is accessible at https://m-creed.github.io/Portfolio-Template/

**Review by iostpa — APPROVED** [review]
Nvm i was wrong, seems to be fine. Ill merge it now

**Pull request merged**